constants/constants.go

```diff
@@ -22,6 +22,7 @@ const (
 const (
 	PAY_INVOICE_SCOPE       = "pay_invoice" // also covers pay_keysend and multi_* payment methods
 	GET_BALANCE_SCOPE       = "get_balance"
+	GET_BUDGET_SCOPE        = "get_budget"
```

Comment:
I am wondering if we should not have a scope for this one - instead it chould always be callable. Is there any benefit of having a scope? (also, an extra scope requires including an extra checkbox in the UI that the user has to understand when configuring permissions)

Reply:
Agreed having it always be callable makes sense. Is there another method which has that behavior right now that I could copy? If not, I'll just dig a bit deeper to make that work.

Reply:
Added an always granted scope with some tests for this. That way, if there are other "always allowed" methods, they can just be added there.

Reply:
I think the `get_info` method also bypasses the check here: https://github.com/getAlby/hub/blob/master/nip47/event_handler.go#L272

We need to come up with a better solution, but for now to see if it works you could try adding an `&& nip47Request.Method != models.GET_BUDGET_METHOD`

Reply:
Oh cool, should I just add get_info to my new ALWAYS_GRANTED scope instead?

Reply:
I'm not sure about the `ALWAYS_GRANTED` scope. This seems to have some side effects (for example, the new scope is now returned and the frontend does not know how to display it) 
![image](https://github.com/user-attachments/assets/91fe8d25-0a53-4703-92c7-aa745c9e6d24)

Also the scope is now included in only new apps in the database (there is a new app_permissions row for always_granted), but only new ones. The older app connections will not have it.

![image](https://github.com/user-attachments/assets/eb889cac-18c1-48a6-bf82-327e28619da7)

And it seems a bit hacky to even call HasPermission explicitly with the always_granted scope which actually just always returns true anyway.

I know my original suggestion is not the best, but on the plus side it's isolated to a single line of code and simply skips the permission check for that request method.

@bumi @im-adithya any thoughts on this one?

Reply:
Ah, thanks for catching the UI issue, I hadn't noticed that! FWIW, my intent here was actually to not store this scope in the DB or show it in the UI at all. Instead, I was hoping to just have it be implicitly granted to all requests. I'll try to make that the case now, but can definitely switch to a specific GET_BUDGET_SCOPE if you prefer. I just think it's a bit weird to show that in the UI in general.

Reply:
Adjusted to avoid always_granted in the db and UI. Let me know if you'd still prefer an explicit budget scope though.

Reply:
Thanks! I also don't think we should have a `GET_BUDGET_SCOPE`. I was just wondering if a single line check like I suggested would be simpler than having this `ALWAYS_GRANTED` scope, and in the end we just bypass the logic anyway.